### ISSUE TITLE
video-to-video?

### ISSUE THREAD

**Issue by videocringe** [HelloVision/HelloMeme#1]
Q: Does HallowMeme support video-2-video mode? Or does it only support the input image?

**Comment by cchance27** [comment]
This was one big addition liveportrait added that was a big selling point if memory serves

**Comment by videocringe** [comment]
> This was one big addition liveportrait added that was a big selling point if memory serves

Yes, it was more like a secondary function in LP, but for me, on the contrary, it is the most important feature, I wonder if there is an analog here or something maybe cooler?

**Comment by songkey** [comment]
I have added examples of image/video generation in the README, hoping to address your questions.

**Comment by inferno46n2** [comment]
> Q: Does HallowMeme support video-2-video mode? Or does it only support the input image?

Just take the first frame of your desired video and use it as the ref image paired with the driving video
